django/utils/autoreload.py

```diff
@@ -285,6 +285,9 @@ def restart_with_reloader():
     while True:
         args = [sys.executable] + ['-W%s' % o for o in sys.warnoptions] + sys.argv
         new_environ = os.environ.copy()
+        if sys.platform == "win32":
```

Comment:
I can't think of a scenario in Python 3 where this would occur without effort to make it happen.  I think it's worth keeping the scope narrow.